src/polyswarm/search.py

```diff
@@ -7,6 +7,11 @@
 from polyswarm_api.types.query import MetadataQuery
 from polyswarm_api.utils import parse_hashes
 
+try:
```

Comment:
This is required because of backwards compatibility with Python 2.7. After adding a test scenario, I relized that JSON validation wasn't working.